### ISSUE TITLE
Possibilidade de post sem sidebar

### ISSUE THREAD

**Issue by anapaulabrasa** [redelivre/guarani#15]
Uma melhoria legal pro Guarani seria poder ativar um uso de posts sem sidebar como há nas páginas (Formato Página sem sidebar). Por mais que se apague os widgets da sidebar, o espaço lateral permanece. Isso quebra a uniformidade dos projetos que usam todas as páginas sem sidebar.

**Comment by eduardozulian** [comment]
Verdade, @anapaulabrasa. Tem algum projeto no ar usando pra eu dar uma olhada?

**Comment by anapaulabrasa** [comment]
Tem sim. Veja aqui, @eduardozulian : http://condicoesdetrabalho.escravonempensar.org.br/2015/06/23/galeria-de-imagens/
Aí dá uma olhada nas páginas do menu.
